src/main/default/lwc/lookup/lookup.js

```diff
@@ -47,7 +49,9 @@ export default class Lookup extends LightningElement {
     }
 
     @api
-    setSearchResults(results) {
+    setSearchResults(results, totalSearchResults) {
+        this._totalSearchResults = totalSearchResults;
```

Comment:
this information should not be passed in again aand again, since it will alway be the same per element.
pass it in via @api attributes and than extend the logic araound it using this infomration